### DISCUSSION TITLE
Minified Lexical error #38

### DISCUSSION THREAD

**Discussion by PunkFleet** [facebook/lexical#4052] [Q&A]
I'm using read-only mode for rendering in production and when I go to the page, I get the error
```
Minified Lexical error #38; visit https://lexical.dev/docs/error?code=38 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```
Lost editorState content after compression
```
{"root":{"children":[],"direction":null,"format":"","indent":0,"type":"root","version":1}}
```
https://github.com/facebook/lexical/issues/2153
I checked out similar issues and it seems this is not working, are there any new fixes or suggestions?

**Comment by PunkFleet** [comment]
When the content of the editor submitted to the database is empty, the load will give an error #38. 
The solution is to validate NotNull in the backend, also in vitejs, maybe using esbuild for minimization instead of terser will solve the problem

**Comment by clementohNZ** [comment]
In my case, it was because I had an invalid editor state json structure. I was switching from a different editor and the previous structure did not match with the structure expected by Lexical.

**Comment by azizbekjuraev** [comment]
> I'm using read-only mode for rendering in production and when I go to the page, I get the error
> 
> ```
> Minified Lexical error #38; visit https://lexical.dev/docs/error?code=38 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
> ```
> 
> Lost editorState content after compression
> 
> ```
> {"root":{"children":[],"direction":null,"format":"","indent":0,"type":"root","version":1}}
> ```
> 
> #2153 I checked out similar issues and it seems this is not working, are there any new fixes or suggestions?

Here is my solution for this:  const content = localStorage.getItem(initialConfig.namespace) || '{"root":{"children":[{"children":[],"direction":null,"format":"","indent":0,"type":"paragraph","version":1}],"direction":null,"format":"","indent":0,"type":"root","version":1}}';
